docs/xks/developer-guide/gitops.md

```diff
@@ -0,0 +1,71 @@
+---
+id: gitops
+title: GitOps a la XKS
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+## What is GitOps?
+
+> GitOps works by using Git as a single source of truth for declarative infrastructure and applications. With GitOps, the use of software agents can alert on any divergence between Git with what's running in [an environment], and if there's a difference, Kubernetes reconcilers automatically update or rollback the cluster depending on the case. &dash; _[Weave Works - Guide To GitOps](https://www.weave.works/technologies/gitops/)_
+
+XKS supports GitHub and Azure DevOps with almost identical workflows. XKF refers to as Git providers. For simplicity, we refer to their CI/CD automation as "pipelines". If you are using GitHub, whenever this text refers to "pipeline", think "GitHub Actions workflow". As you saw in the previous section, XKS comes with a set of pipelines that automatically detects app releases and promotes them through a series of environments. The allows both rapid iteration and strong validation of apps.
+
+XKS is built around [trunk-based development](https://trunkbaseddevelopment.com/).
+
+## Emilia updates an app
```

Comment:
Should we say something like "User story: Emilia updates an app" ?

Reply:
Fixed.